### PR TITLE
feature/ads-1054 Add skiplink and bsc-logo tokens

### PR DESCRIPTION
**Pull request by cblva**

### CODE REVIEW

File: src/tokens/corporate/dark.json [hunk 1]
```diff
@@ -184,6 +184,15 @@
     },
     "accent-featured-tag": {
       "value": "{accent-primary colors.accent-primary-80}"
+    },
+    "bsc-logo": {
+      "value": "{neutral colors.neutral-00}"
+    },
+    "bsc-logo-tagline": {
+      "value": "{neutral colors.neutral-00}"
+    },
```
Comment (kaseyllabelle) [inline src/tokens/corporate/dark.json]: these should be neutral-100 in dark

File: src/tokens/corporate/globals/colors.json [hunk 2]
```diff
@@ -3,6 +3,12 @@
     "type": "color",
     "brand-primary": {
       "value": "#003c71"
+    },
+    "bsc-logo-primary": {
+      "value": "#00467F"
+    },
+    "bsc-logo-secondary": {
+      "value": "#6A737B"
```
Comment (kaseyllabelle) [inline src/tokens/corporate/globals/colors.json]: can we lowercase the hex values here. let's also check the linters are working as expected. i thought the autofix was set up to update those.

File: src/tokens/corporate/dark.json [hunk 3]
```diff
@@ -184,6 +184,15 @@
...
+    "surface-skiplink": {
+      "value": "{neutral colors.neutral-95}"
```
Comment (kaseyllabelle) [inline src/tokens/corporate/dark.json]: this should be neutral-20

File: src/tokens/watchman/dark.json [hunk 4]
```diff
@@ -184,6 +184,15 @@
     },
     "accent-featured-tag": {
       "value": "{neutral colors.neutral-100}"
+    },
+    "bsc-logo": {
+      "value": "{neutral colors.neutral-00}"
+    },
+    "bsc-logo-tagline": {
+      "value": "{neutral colors.neutral-00}"
+    },
```
Comment (kaseyllabelle) [inline src/tokens/watchman/dark.json]: these should be neutral-100

File: src/tokens/watchman/globals/colors.json [hunk 5]
```diff
@@ -3,6 +3,12 @@
     "type": "color",
     "brand-primary": {
       "value": "#e57200"
+    },
+    "bsc-logo-primary": {
+      "value": "#00467F"
+    },
+    "bsc-logo-secondary": {
+      "value": "#6A737B"
```
Comment (kaseyllabelle) [inline src/tokens/watchman/globals/colors.json]: same re: casing

**Review by APJohns — APPROVED** [review]

**Pull request merged**